docs/v2/index.md

```diff
@@ -1,5 +1,7 @@
 ### [Click here for the full API documentation](https://shaid.smartdevicelink.com/docs)
-# SHAID v2.5.0
+
+SHAID v2.6.0+ now supports issue tracking and facilitating OEM communication with App Developers. Click [here](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0218-developer-oem-communication.md) for the full proposal.
```

Comment:
```suggestion
```
I think this makes more sense to include in the SHAID 2.6 release notes, than this API Documentation.